### PR TITLE
Add contribution rules

### LINKED ISSUE #6: Add contribution rules

### PR DESCRIPTION
**Pull request by mahasak**
I have add contributtion rule as CONTRIBUTING.md to help us maintain our repository.
- fix #6 

Please review and merge to master

### CODE REVIEW

**Comment by varokas** [comment]
In general. I think this is way too long. At least I suggest make the "How to contribute" section comes first with a few simple rules on how to contribute and how the repo owner will manage 

Eg. 

How to contibute
* Please +1 the comment you like rather than post "me too"
* Please comment LGTM if you read the PR and it is good to you so the contributor knows
...
...

Then the next sections can be the boring legal stuff

**Comment by varokas** [comment]
Also if you think it's good to you, please comment LGTM so that i know.

**Comment by mahasak** [comment]
Should i split other section to other MD ?
for example LICENSE.md, COC.md, HowToSubmitPR.md ?

File: CONTRIBUTING.md
Comment (iboss-ptk) [inline CONTRIBUTING.md]: I think revert format should be mentioned here as one in the list of all possibilities of the type of commit message.

And I expect the definition of `<type>`, `<scope>` etc. to be right after the template or else I would think that it does not exist (it took me some time to find out).

And I think example would be valuable.

File: CONTRIBUTING.md
Comment (iboss-ptk) [inline CONTRIBUTING.md]: This part is not quite clear as I tried to understand. We might need to articulate that how it's gonna look like with and without breaking changes.

File: CONTRIBUTING.md
Comment (iboss-ptk) [inline CONTRIBUTING.md]: This looks like a link to the commit message part in this document but it doesn't work since there is no `<a name='commit-message-format'></a>` up there and it should look like this `[document](#commit-message-format)` if I understand your intention correctly.

**Review by iboss-ptk — CHANGES_REQUESTED** [review]
There are few things that are quite ambiguous. And I think examples would be valuable.

And as discussed above, I think separate CoC, how to contribute and License from each other would be a good idea. Another thing I want to point out is the sequence of topics which I think it doesn't reflect the sequence of the contribution. I would suggest:

- Submitting Issues and Bugs
- Feature Requests
- Coding Rules
- Commit Message Guidelines
- Submitting Pull request
- Got a Question or Problem?

File: CONTRIBUTING.md
Comment (mahasak) [reply CONTRIBUTING.md]: I am going to made some change on CONTRIBUTIONG.md per @iboss-ptk & @varokas comments, Thanks mate 😄

**Comment by mahasak** [comment]
@iboss-ptk  @varokas I'd manage to change regarding the comments, If you guys review and ok with it LGTM !!!!

**Comment by varokas** [comment]
LGTM. 

Maybe we should add 2 more things according to this: https://www.facebook.com/mahasak.pijittum/posts/10214883212934174?notif_t=like_tagged&notif_id=1504938323613925

1. Communicate in English
2.Pull Requests should have tests accompanied the code changes

**Comment by varokas** [comment]
I think we should have some rules about ( how long we need to wait || how many LGTM to get ) before merged.

**Comment by iboss-ptk** [comment]
I would be good to explain what LGTM means (looks good to me?). For new contributors, this might be confusing.

**Comment by mahasak** [comment]
@iboss-ptk LGTM -> Looks good to me, Let's get that moving, Let's get this merged :)
Already add to markdown.

**Comment by varokas** [comment]
I will merge this tonight if theres no other comments

**Comment by iboss-ptk** [comment]
LGTM!

**Pull request merged**
